axelor-account/src/main/java/com/axelor/apps/account/service/AccountService.java

```diff
@@ -207,4 +211,13 @@ protected Account desactivate(Account account) {
     account.setStatusSelect(AccountRepository.STATUS_INACTIVE);
     return account;
   }
+
+  @Transactional
+  public void updateMoveLineServiceType(Account account) {
+    moveLineRepository
+        .all()
+        .filter("self.account = :account")
+        .bind("account", account)
+        .update("serviceType", account.getServiceType());
+  }
```

Comment:
We can't update all account MoveLines like that. We can have serveral records. Period and JPA cache is not managed, we can't update some old move lines.